beacon-chain/utils/shuffle.go

```diff
@@ -145,6 +113,19 @@ func PermutedIndex(index uint64, listSize uint64, seed [32]byte) (uint64, error)
 		if bitV == 1 {
 			index = flip
 		}
+		if dir {
```

Comment:
How about naming this `shuffle`? Then you don't need to explain that the truth case is shuffle and the false case is unshuffle in a comment.

Reply:
good suggestion , changed